src/helpers/_styleguide-example.scss

```diff
@@ -0,0 +1,80 @@
+/*
+ * _styleguide-example.scss
+ *
+ * This is an example file that adheres to our .stylelintrc rules and style.
+ */
+
+@import url('x.css');
+@import url('y.css');
```

Comment:
No, I don't think we should, I'll remove this. Unfortunately stylelint doesn't seem to have a rule to disallow url @imports.